src/Http/Controllers/TimoneiroBaseController.php

```diff
@@ -34,9 +34,9 @@ public function index(Request $request)
         $query = $model->query();
 
         if ($search) {
-            $searchable = array_keys($dataType->field_set);
+            $searchable = $dataType->list_display;
```

Comment:
Don't rely on the `list_display` attribute, coz it might contain calculated fields (eloquent accessors). I think the best thing is to have search fields defined separately in the config file. What do you think?